packages/react-magma-dom/src/components/NativeSelect/NativeSelect.test.js

```diff
@@ -159,5 +173,36 @@ describe('NativeSelect', () => {
         'flex'
       );
     });
+
+    it(`Shouldn't display an additional wrapper when additionalContent is unset'`, () => {
```

Comment:
I think this would be better as 2 separate tests, no rerenders. 
1. When additionalContent is present, a wrapper is present
2. When there's no additionalContent present, a wrapper is NOT present